sample_sandbox/dev.jl

```diff
@@ -0,0 +1,14 @@
+using WGLMakie, GeoMakie, GraphMakie, Bonito
```

Comment:
so far we haven't committed sandbox directories. So should this be in documentation instead?

Reply:
I'll just remove for now - since you guys have a working practise around it I think it is well enough documented already/in the new Docker docs.